dashboard/app/models/lesson.rb

```diff
@@ -297,7 +297,8 @@ def summarize_for_lesson_edit
       announcements: announcements,
       activities: lesson_activities.map(&:summarize_for_edit),
       resources: resources,
-      objectives: objectives.map(&:summarize_for_edit)
+      objectives: objectives.map(&:summarize_for_edit),
+      course_version_id: lesson_group.script.course_version&.id
```

Comment:
Can you please change `course_version_id` to `courseVersionId` here (and in the client code which receives it)? We're not great about this, but the direction we're trying to head in is to convert to camelCase on the server, so that the client doesn't see any snake_case. this opens the possibilities for a few ways to do automatic case conversion on the server in the future.